requirements/base.txt

```diff
@@ -13,6 +13,7 @@ django-waffle==0.11.1
 djangorestframework==3.2.3
 djangorestframework-jwt==1.8.0
 drf-extensions==0.2.8
+python-social-auth>=0.2.19,<0.3.0
```

Comment:
It's not explicitly, but is a requirement of edx-auth-backends, that is just below this one in the file. In version 0.5.0 of edx-auth-backends, this was requiring python-social-auth>=0.2.19,<1.0.0, but versions of python-social-auth greater that 0.3.0 break the tests. In version 0.6.0 of edx-auth-backends it is fixed, using python-social-auth>=0.2.19,<0.3.0 (the line I've added), but I didn't update edx-auth-backends to 0.6.0 because I don't know if it's compatible with our ecommerce version, in fact I didn't test that option